README.md

```diff
@@ -89,6 +89,8 @@ $ brew install git automake autoconf libtool unbound
 You're a Linux user so you probably already know what to do. Make sure you have
 git, autotools, libtool, and unbound installed via whatever package manager
 your OS uses.
+You can install these dependencies using
+`sudo apt install -y autotools-dev libtool libunbound-dev`
```

Comment:
The GitHub Actions CI only installs libunbound-dev: https://github.com/handshake-org/hnsd/blob/master/.github/workflows/test.yml#L25

Are the others installed by default on Ubuntu?

Also maybe worth mentioning that this is for Debian/Ubuntu/etc. and not for all Linux.

Reply:
Yeah I needed to install the rest of them on one of my vms. So I figured better sure than sorry